android_jni/avifandroidjni/src/main/jni/CMakeLists.txt

```diff
@@ -35,7 +35,8 @@ add_library("avif_android" SHARED "libavif_jni.cc")
 
 # Import the cpu-features module to compute the number of threads used for
 # decoding.
-include(AndroidNdkModules)
-android_ndk_import_module_cpufeatures()
+set(CPU_FEATURES_DIR "${ANDROID_NDK}/sources/android/cpufeatures")
+include_directories(${CPU_FEATURES_DIR})
+add_library(cpufeatures STATIC "${CPU_FEATURES_DIR}/cpu-features.c")
```

Comment:
is there any reason to build this file from source explicitly?

what was wrong with the current way of simply importing `AndroidNdkModules`?

Reply:
I'm pretty sure that was removed in r22 and above, and r25c won't compile with it, with an error stating that AndroidNdkModules can't be found.